src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -120,6 +120,26 @@ def get(self, svc_id):
             'osd_metadata': mgr.get_metadata('osd', svc_id),
         }
 
+    @RESTController.Resource('GET')
+    @handle_send_command_error('osd')
+    def histogram(self, svc_id):
+        """
+        :return: Returns the requested data. The `histogram` key may contain a
+                 string with an error that occurred if the OSD is down.
+        """
+        try:
+            histogram = CephService.send_command(
+                'osd', srv_spec=svc_id, prefix='perf histogram dump')
+        except SendCommandError as e:  # pragma: no cover - the handling is too obvious
+            if 'osd down' in str(e):  # pragma: no cover - no complexity there
+                histogram = str(e)
+            else:  # pragma: no cover - no complexity there
+                raise
+
+        return {
```

Comment:
I don't think it is necessary to wrap this in an object. When i call this endpoint i expect the data directly and not wrapped in a dict.

```suggestion
        return histogram
```